.github/workflows/build.yml

```diff
@@ -53,14 +53,55 @@ jobs:
       - name: Build
         run: yarn build
 
-      - name: Publish Docs
-        if: ${{ github.ref == 'refs/heads/master' }}
-        run: |
-          yarn docs --quiet
-          yarn firebase deploy --token ${{secrets.FIREBASE_TOKEN}}
-
       - name: Publish
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         run: |
           echo "//registry.npmjs.org/:_authToken=${{secrets.NPM_TOKEN}}" > ./.npmrc
           yarn lerna publish from-git --yes
+
+  build-docs:
```

Comment:
why do we need separate step?

Reply:
So we do not wait for builds

Reply:
but it's harder to maintain workflow file now

Reply:
Can we reuse those setup steps? If not I will revert them